src/gc-common.h

```diff
@@ -43,6 +68,14 @@ extern jl_gc_callback_list_t *gc_cblist_notify_external_alloc;
 extern jl_gc_callback_list_t *gc_cblist_notify_external_free;
 extern jl_gc_callback_list_t *gc_cblist_notify_gc_pressure;
 
+
+// FIXME: These are specific to the Stock GC but being declared here
+// for now, instead of gc-stock.h. We might want to refactor the
+// code in gc-stacks.c that uses these
+extern _Atomic(int) gc_ptls_sweep_idx;
+extern _Atomic(int) gc_stack_free_idx;
+extern _Atomic(int) gc_n_threads_sweeping_stacks;
+
```

Comment:
These are helper variable used by the stock GC stack pool sweeping algorithm. They should be moved to the stock GC.

Also, we should move the function to sweep stack pools into the interface, even though they will only be used by the GC and not other code in the runtime.